libs/blocks/marquee/marquee.js

```diff
@@ -161,8 +161,16 @@ export default function init(el) {
       media.classList.add('bleed');
       foreground.insertAdjacentElement('beforebegin', media);
     }
-    if (media?.lastChild.textContent.trim()) {
-      const mediaCreditInner = createTag('p', { class: 'body-s' }, media.lastChild.textContent);
+
+    let mediaCreditInner;
+    const txtContent = media?.lastChild.textContent.trim();
+    if (txtContent) {
+      mediaCreditInner = createTag('p', { class: 'body-s' }, txtContent);
+    } else if (media.lastElementChild && media.lastElementChild.tagName !== 'PICTURE') {
```

Comment:
Maybe shorten this to just `media.lastElementChild?.tagName !== 'PICTURE'`?

Reply:
Adressed, thank you.